index.js

```diff
@@ -1,18 +1,43 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'purple'
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
 
+document.querySelector('div#profile h2').innerText = traveler.name
+document.querySelector('div#profile em').innerText = traveler.nickname
+document.querySelector('div#profile img').src = traveler.photo
+document.querySelector('div#profile img').alt = traveler.name
+
+
```

Comment:
I like the way you chained the attributes to update! In other cases where you might be working with more than one of the same css attribute, you might want to save them to diff variables and update them accordingly, but your logic is correct for this! Awesome job!